lib/bowling_scorecard.rb

```diff
@@ -0,0 +1,179 @@
+require_relative 'frame'
+
+class BowlingScorecard
```

Comment:
Super massive class. Probably could be divided into smaller classes as you said in README